Wireframe/css/style.css

```diff
@@ -0,0 +1,67 @@
+body{
+    font-family: sans-serif;
+    font-family: 'Libre Baskerville', serif;
+    margin: 20px 100px;
+    background-color: rgb(233, 237, 240);
+ 
+    
+}
+
+header {
+   margin-bottom: 50px;
+}
+
+h1 {
+    font-size: 5rem;
+    text-align: center;
+}
+.article1 {
+    border: 2px black solid;
+    margin-bottom: 30px;
+    display: flex;
+    height: auto;
+    width: auto;
+}
+
+
+article{
```

Comment:
This is not incorrect, but with elements like article, section, etc where it is expected that you might have more than one per page, it is worth using class selectors `.article` rather than an element selector - this is to prevent awkward styling bugs appearing later when you create an article element that you want to be styled differently